db/data/20230612081755_set_correct_receiving_organisations.rb

```diff
@@ -60,6 +60,6 @@
   actual.save!
   actual.reload
 
-  puts "BEFORE: Actual #{actual.id} now has recipient organisation: #{actual.recipient_organisation_name}"
+  puts "AFTER: Actual #{actual.id} now has recipient organisation: #{actual.recipient_organisation_name}"
```

Comment:
Let's squash this commit into the previous one as I don't think it necessarily needs documenting as its own change.